docker/apache-ignite-net/Dockerfile

```diff
@@ -29,4 +29,7 @@ WORKDIR /app
 COPY libs ./libs
 COPY publish ./
 
+# Container port exposure
+EXPOSE 47500
```

Comment:
Should we use same ports as Java-based Dockerfile does?
`EXPOSE 11211 47100 47500 49112`